package.json

```diff
@@ -44,10 +44,17 @@
     "@tensorflow-models/body-segmentation": "^1.0.1",
     "@tensorflow-models/face-landmarks-detection": "1.0.5",
     "@tensorflow-models/hand-pose-detection": "^2.0.0",
+    "@tensorflow-models/mobilenet": "^2.1.0",
     "@tensorflow-models/pose-detection": "^2.1.0",
     "@tensorflow/tfjs": "^4.2.0",
     "@tensorflow/tfjs-vis": "^1.5.1",
     "axios": "^1.3.4",
     "webpack-merge": "^5.9.0"
+},
+  "overrides": {
```

Comment:
We explored "overrides" before @ziyuan-linn made us switch to `yarn`, where this doesn't seem to be necessary anymore. Probably good to delete this, especially if it installs on yarn without it.